### ISSUE TITLE
iOS 11 issue?

### ISSUE THREAD

**Issue by bland328** [brutella/hklifx#34]
Under iOS 11, when I use either the Apple "Home" app or the brutella "Home" app to attempt to add an accessory with hklifx running, no LIFX bulbs are found.

Furthermore, hklifx sees no inquiries at all. The only feedback I ever get from hklifx right now is:

`DEBUG 2017/09/21 17:57:40 Discovered Group Bedroom`
`DEBUG 2017/09/21 17:57:40 Discovered Location My Home`
`DEBUG 2017/09/21 17:57:40 Discovered Device Right`
`DEBUG 2017/09/21 17:57:40 Creating New HKLight for Right`
`INFO 2017/09/21 17:57:40 Accessory IP is 192.168.1.239`
`DEBUG 2017/09/21 17:57:40 Discovered Device Left`
`DEBUG 2017/09/21 17:57:40 Creating New HKLight for Left`
`INFO 2017/09/21 17:57:40 Accessory IP is 192.168.1.239`

I can't find any activity that inspires even one more message out of hklifx.

Barring ongoing memory leak problems, hklifx used to work for me, though I don't know for certain that iOS 11 changes are to blame for this.

Is anyone else having the same _or_ contradictory results under iOS 11?

**Comment by bland328** [comment]
I'm going to leave this here for now, in case it helps someone, but I did get hklifx working again, and here's how:

1. I deleted "My Home" from the Apple Home app, and recreated it, but saw no difference.
2. I killed hklifx, deleted the subfolders it had created with names that matched my bulb names, then restarted it.

Now, I'm able to add my LIFX bulbs again. I don't know whether I needed to do both of these things, or only one of them. Perhaps step 1 fixed it, but I wasn't patient enough.
